bson/src/test/unit/org/bson/codecs/EnumCodecProviderTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bson.codecs;
+
+import org.bson.codecs.configuration.CodecRegistries;
+import org.bson.codecs.pojo.entities.SimpleEnum;
```

Comment:
It's not a new class from this PR, so I didn't think to move it.  What package do you think it should be moved to?  `org.bson.codecs.entities`, perhaps?  This would be a new package, if so.

Reply:
Yeah, or just `org.bson.codecs` as there is only going to one enum tested.

Reply:
Done